src/main/java/net/praqma/tracey/broker/impl/rabbitmq/TraceyRabbitMQReceiverImpl.java

```diff
@@ -1,45 +1,38 @@
 package net.praqma.tracey.broker.impl.rabbitmq;
 
-import com.rabbitmq.client.AMQP;
-import com.rabbitmq.client.AlreadyClosedException;
-import com.rabbitmq.client.Channel;
-import com.rabbitmq.client.Consumer;
-import com.rabbitmq.client.DefaultConsumer;
-import com.rabbitmq.client.Envelope;
-import com.rabbitmq.client.ShutdownSignalException;
+import com.rabbitmq.client.*;
+import net.praqma.tracey.broker.api.TraceyFilter;
+import net.praqma.tracey.broker.api.TraceyIOError;
+import net.praqma.tracey.broker.api.TraceyReceiver;
+
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
 import java.util.concurrent.TimeoutException;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-import net.praqma.tracey.broker.api.TraceyFilter;
-import net.praqma.tracey.broker.api.TraceyIOError;
-import net.praqma.tracey.broker.api.TraceyReceiver;
-
 /**
  * <h2>Default RabbitMQ receiver implementation</h2>
  * <p>
  * Basic implementation for RabbitMQ. Prints received messages to console.
  * </p>
  */
-public class TraceyRabbitMQReceiverImpl implements TraceyReceiver <RabbitMQRoutingInfo> {
+public class TraceyRabbitMQReceiverImpl implements TraceyReceiver<RabbitMQRoutingInfo> {
 
     private static final Logger LOG = Logger.getLogger(TraceyRabbitMQReceiverImpl.class.getName());
-    private RabbitMQConnection connection;
+
     private TraceyRabbitMQMessageHandler handler;
```

Comment:
If you started to convert fields to final then stay consistent and convert handler too.